moabb/paradigms/motor_imagery.py

```diff
@@ -68,23 +74,20 @@ def process_raw(self, raw, dataset):
                                include=channels)
 
         # get event id
-        if self.events is None:
-            event_id = dataset.event_id
-        else:
-            event_id = {ev: dataset.event_id[ev] for ev in self.events}
+        event_id = self.used_events(dataset)
 
         # pick events, based on event_id
         try:
             events = mne.pick_events(events, include=list(event_id.values()))
-        except RuntimeError:
+        except RuntimeError as r:
             # skip raw if no event found
             return
 
         # get interval
         if self.interval is None:
             tmin, tmax = dataset.interval
         else:
-            tmin, tmax = self.interval
+            tmin, tmax = [t + dataset.interval[0] for t in self.interval]
```

Comment:
We should add a something to define only one part of the interval.
for example `[1, None]` will just start from 1 to the end of the MI period.
This usecase allow to cut out the cue that usually contain a lot of predictive power and bias the results.

Reply:
What do you think of changing the input parameter `interval` to `tmin` and `tmax`. I can make the change

Reply:
yes actually thats easier and you're right about the None. It would be great if you did that :)

Reply:
okay, im doing it